<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addproductwindow</class>
 <widget class="QDialog" name="addproductwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addProduct">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
background-color: darkblue; 
color: white;</string>
   </property>
   <property name="text">
    <string>Добавить товар</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
background-color: darkred; 
color: white;</string>
   </property>
   <property name="text">
    <string>Вернуться в главное меню</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>691</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;
</string>
      </property>
      <property name="text">
       <string>Название магазина</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="shopLine">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
      <property name="text">
       <string>Адрес магазина</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="addresLine">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
      <property name="text">
       <string>Название товара</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="productLine">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
      <property name="text">
       <string>Стоимость</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="priceLine">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
      <property name="text">
       <string>Количество</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="quantLine">
      <property name="styleSheet">
       <string notr="true">font-family: Arial, sans-serif;
font-size: 20px;
color: darkblue;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
